src/main/java/eu/dissco/core/datacitepublisher/domain/datacite/DcPublisher.java

```diff
@@ -0,0 +1,11 @@
+package eu.dissco.core.datacitepublisher.domain.datacite;
+
+import lombok.Value;
+
+@Value
+public class DcPublisher {
+  String name = "Distributed System of Scientific Collections";
```

Comment:
Wondering if we need to make these properties with a default value. Probably not as they probably won't change

Reply:
I thought about that, but then wouldn't we need to inject a properties bean into this class? I think it would get messy

Reply:
Yeah you would then have to build this object and in the director/service have then bean. Let's leave it like this for now we can always change it.